packages/frontend-shared/src/gql-components/StatusBadge.vue

```diff
@@ -67,44 +32,4 @@ const props = defineProps<{
   isApp: boolean
 }>()
 
-const { t } = useI18n()
-
-const emit = defineEmits<{
-  (event: 'chooseABrowser'): void
-}>()
-
-gql`
-mutation TestingTypeSelectionAndReconfigure($testingType: TestingTypeEnum!, $isApp: Boolean!) {
-  setTestingTypeAndReconfigureProject(testingType: $testingType, isApp: $isApp) {
-    currentProject {
-      id
-      currentTestingType
-      isCTConfigured
-      isE2EConfigured
-      isLoadingConfigFile
-      isLoadingNodeEvents
-    }
-  }
-}
```

Comment:
Should we remove the code on the graphql side as well? it seems that the mutation `setTestingTypeAndReconfigureProject` is only being used from this component